pull_request_template.md

```diff
@@ -1,65 +1,24 @@
 
 # Test instructions | Instructions pour tester la modification
 
-> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
-> will help a developer test things out without too much detective work. Also,
-> include any environmental setup steps that aren't in the normal README steps
-> and/or any time-based elements that this requires.
-
----
-
-> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
-> modification. Elles aideront les développeurs à faire des tests sans avoir à
-> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
-> de l’environnement qui ne font pas partie des étapes normales dans le fichier
-> README et tout élément temporel requis.
+_TODO: Fill in test instructions for the reviewer._
 
 # Release Instructions | Instructions pour le déploiement
 
 None.
 
-> Necessary steps to perform before and after the deployment of these changes.
-> For example, emptying the cache on a feature that changes the cache data
-> structure in Redis could be mentioned.
-
----
-
-> Étapes nécessaires à exécuter avant et après le déploiement des changements
-> introduits par cette proposition. Par exemple, vider la cache suite à des
-> changements modifiant une structure de données de la cache pourrait être
-> mentionné.
-
 # Reviewer checklist | Liste de vérification du réviseur
 
-This is a suggested checklist of questions reviewers might ask during their
-review | Voici une suggestion de liste de vérification comprenant des questions
-que les réviseurs pourraient poser pendant leur examen :
-
-
-- [ ] Is the code maintainable? | Est-ce que le code peut être maintenu?
-- [ ] Have you tested it? | L’avez-vous testé?
-- [ ] Are there automated tests? | Y a-t-il des tests automatisés?
-- [ ] Does this cause automated test coverage to drop? | Est-ce que ça entraîne
-      une baisse de la quantité de code couvert par les tests automatisés?
-- [ ] Does this break existing functionality? | Est-ce que ça brise une
-      fonctionnalité existante?
-- [ ] Does this change the privacy policy? | Est-ce que ça entraîne une
-      modification de la politique de confidentialité?
-- [ ] Does this introduce any security concerns? | Est-ce que ça introduit des
-      préoccupations liées à la sécurité?
-- [ ] Does this significantly alter performance? | Est-ce que ça modifie de
-      façon importante la performance?
-- [ ] What is the risk level of using added dependencies? | Quel est le degré de
-      risque d’utiliser des dépendances ajoutées?
-- [ ] Should any documentation be updated as a result of this? (i.e. README
-      setup, etc.) | Faudra-t-il mettre à jour la documentation à la suite de ce
-      changement (fichier README, etc.)?
+- [ ] This PR does not break existing functionality.
+- [ ] This PR does not violate GCNotify's privacy policies.
+- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.

```

Comment:
Can we link to the Risk Register doc here 🙏

Reply:
hmmm... what is the likelihood that a reviewer is going to click and scroll through that document each time? I'd err on not having it in the PR review

Reply:
I thought of linking it but I am paranoid. I checked access to the document and this is protected, but I fell for security through obscurity on this one as this is a sensitive document. I can put it there if you pay me cider later on.